Gaijinizer patch/Map070.txt

```diff
@@ -1702,98 +1764,99 @@ method.#MTLed
 意外にハルを抱く人間が多くて、ボロかったよ。
  驚きだったね。
 > CONTEXT: Map070/events/9/pages/1/47/Dialogue/enemy7
-I laughed at first that it was laughable to get
-along with humans. There were a lot of people who
-had Haru unexpectedly, and it was tedious.
-I was surprised.#MTLed
+At first I laughed at the idea of getting along
+with humans, it sounded stupid. But there were a
+surprising number of humans who'd fuck Haru, and
+it was pretty profitable. Very surprising.#Ron
```

Comment:
Let's break this up into two text boxes so we don't have to squish this all together...
>_**At first, I laughed at the idea of getting
along with humans...**_
>
>_**It sounded stupid.
But there were a surprising amount of humans
who wanted to fuck Haru, and it was quite profitable.**_
>
>_**I was quite surprised.**_